_pages/algorithm-development/languages/python.md

```diff
@@ -10,57 +10,90 @@ image:
+def apply(input, globals):
+    return "hello {} {}".format(str(input), str(globals['payload']))
 
-<img src="{{site.cdnurl}}{{site.baseurl}}/images/post_images/algo_dev_lang/dependencies_python.png" alt="Python Dependency File" class="screenshot img-md">
+def load():
+    globals = {}
+    globals['payload'] = "Loading has been completed."
+    return globals
 
-If you have any dependencies you can add them by typing in the package name to the `requirements.txt` file.
+algorithm = ADK(apply, load)
+algorithm.init("Algorithmia")
+{% endhighlight %}
 
-This guide won't depend on any external dependencies so you can close the dependencies window.
+In our load function we've created a `globals` object and added a key called `payloads` with a string value `Loading has been completed.`; we then return the globals object, which will be passed as input to the algorithm's `apply` function, where we use that value as part of the algorithm's output. Executing the algorithm locally will result in `Hello Algorithmia Loading has been completed.` being printed to `stdout`.
 
-If you do add dependencies, you will still need to import those packages via the import statement to your algorithm file as you would do for any Python script.
-{: .notice-info}
+## Available Libraries
+
+In addition to your own code in `src/Algorithm.py`, Algorithmia makes a number of libraries available to make algorithm development easier. We support multiple versions of Python and a variety of frameworks, and we continue to add new variants and broaden GPU support. A complete list of supported environments can be found on the [Enviornment Matrix](/developers/model-deployment/environments/) page, and are available through the "Environment" drop-down when creating a new algorithm.
+
+<img src="{{site.cdnurl}}{{site.baseurl}}/images/post_images/algo_dev_lang/env_dropdown_python.png" alt="Algorithm creation modal, Environment drop-down" class="screenshot">
+
+You can utilize common Python libraries such as <a href ="{{site.baseurl}}/model-deployment/scikit/">Scikit-learn</a>, <a href ="{{site.baseurl}}/model-deployment/tensorflow/">Tensorflow</a>, Numpy and many others by adding them as a dependency in your algorithm.
+
+Also, algorithms can call other algorithms and manage data on the Algorithmia platform. You can learn more about calling algorithms in the <a href="{{site.baseurl}}/clients/python">Algorithmia Python Client Guide</a>.
+
+## Managing Dependencies
+
+Algorithmia supports adding 3rd party dependencies via the <a href="https://pypi.python.org/pypi">Python Package Index (PyPI)</a> using a `requirements.txt` file. Add any dependencies you have by typing the package name into the `requirements.txt` file. If you do add dependencies, you will still need to import those packages via the import statement to your algorithm file as you would do for any Python script.
```

Comment:
3rd party --> third-party ; 
suggest shortening: "using a `requirements.txt` file. Add any dependencies you have by typing the package name into the `requirements.txt` file." --> "using a `requirements.txt` file, where you can add any dependencies you have."